services/scripts.py

```diff
@@ -1,61 +1,66 @@
 # Music recommendation application
 import csv
 
-from cfg import TableEnum
+from cfg import TableIndexes
 
 
 class Song:
     def __init__(self, row: list[str]) -> None:
-        self.id = int(row[TableEnum.ID.value])
-        self.name = row[TableEnum.Track.value]
-        self.album = row[TableEnum.Album.value]
-        self.artist = row[TableEnum.Artist.value]
+        self.id = row[TableIndexes.ID.value]
+        self.name = row[TableIndexes.TRACK.value]
+        self.album = row[TableIndexes.ALBUM.value]
+        self.artist = row[TableIndexes.ARTIST.value]
 
 
 class DataBase:
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> None:
         self.db = {}
 
         with open(filename, encoding='utf-8-sig') as file:
             _ = file.readline()
             reader = csv.reader(file)
 
             for row in reader:
-                self.add_song(Song(row))
+                self.add_song(self.__parse_row(row))
 
-    def add_song(self, song: Song) -> None:
-        self.db[song.id] = song
+    @staticmethod
+    def __parse_row(row: list[str]) -> Song:
+        row[TableIndexes.ID.value] = int(row[TableIndexes.ID.value])
 
-    def get_song(self, song_id: int) -> Song:
-        return self.db.get(song_id)
+        return Song(row)
 
+    def add_song(self, song: Song) -> None:
+        if song.id in self.db:
+            raise Exception('A song with this id already exists!')
```

Comment:
```suggestion
            raise ValueError('A song with this id already exists!')
```
I think it should be a `ValueError` (as a bad value, idk), maybe not, but it shouldn't be a simple `Exception`